src/index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+function moveFile() {
+  const fileToMove = process.argv[2];
+  const selectedPath = process.argv[3];
+
+  if (!fs.existsSync(fileToMove)) {
+    console.error(`${fileToMove} does not exist`);
+
+    return;
+  }
+
+  const content = fs.readFileSync(fileToMove);
+
+  if (fs.existsSync(selectedPath)) {
+    fs.stat(selectedPath, (error, stats) => {
+      if (error) {
+        throw new Error(error);
+      }
+
+      if (stats.isDirectory()) {
+        fs.writeFileSync(
+          `${selectedPath}/${fileToMove}`,
+          content,
+          (err) => {
+            if (err) {
+              throw new Error(error);
+            }
+          },
+        );
```

Comment:
you're using the same function several times. try to create the variable and change it in `if` construction. then use this `writeFileSync` at the end